### PR TITLE
JS Day 1 Exercises 1-7

### PR DESCRIPTION
**Pull request by RupySB**

### CODE REVIEW

File: workshop/exercise-7.js [hunk 1]
```diff
@@ -6,6 +6,14 @@
 // Eat at 7h, 13h and 18h
 // The rest of the time is spent training.
 
+for (let time= 0; time< 24; time++) {
+    let job= 'train';
```
Comment (SSaquif) [inline workshop/exercise-7.js:10]: u have named your variable job here but while printing it you have called it activity, so there's error in the code. As activity has never been defined. Please always run your code and test if it's actually working.

File: workshop/exercise-7.js [hunk 2]
```diff
@@ -6,6 +6,14 @@
...
+    if (time === 7 || time=== 13 || time=== 18) {
+        job= 'eat';}
+    if (time< 6 || time> 21) {
+    job= 'sleep';}
+    console.log(`${time} to do this ${activity}`);
```
Comment (SSaquif) [inline workshop/exercise-7.js:15]: activity variable is undefined

**Comment by SSaquif** [comment]
Sorry could not give marks for Q7 as the code doesn't run, but your logic is good. Please correct it and resubmit.

**Comment by SSaquif** [comment]
Also It does seem make it seem like you copied from the solution and forgot to change the variable name. If you are having issues pls know that you can use the queue multiple times and each time we could build up little by little on your issues if you are having a hard time.

**Comment by RupySB** [comment]
Yes, thank you! I have a list of questions I wrote down and Im definitely going to go over them after the lecture! (:

________________________________

**Comment by RupySB** [comment]
Hi,

I fixed up #7,  I just spoke to Richard & he wasn't sure how I can update it there, but I pushed it on my end. Also, I'll add the code here:

for (let time = 0; time < 24; time++) {
  let job = "train";
  if (time > 21 || time < 6) {
    job = "sleep";
  }
  if (time === 7) {
    job = "eat";
  } else if (time === 13) {
    job = "eat";
  } else if (time === 18) {
    job = "eat";
  }
  console.log(`${time} You need to  ${job}`);
}



________________________________

**Pull request closed without merging**